### ISSUE TITLE
Typos in training data

### ISSUE THREAD

**Issue by juand-r** [wangcunxiang/SemEval2020-Task4-Commonsense-Validation-and-Explanation#4]
I saw a couple typos in the training data:

id 10: "Eggs eat kis on Easter." kis -> kids
id 332: "He read a wardrobe in rhe table"  rhe -> the
id 708: "He cooked sphegatti in his kitchen"
id 995: "He bouhgt a laptop at the ice cream store."
id 1099: "He eats a box of ceral"
id 1145: "A minister is part of the governement."
id 1228: "He put a frozen pizza in the refridgerator"
id 1240: "I put dirty utensils in the wasing machine." wasing -> washing
id 1341: "he took the boat to croos the street" croos -> cross
id 1643: "My dad eats a pineaple every day in evening"
id 1658 "The doctor adviced him to keep smoking to stay healthy." adviced him -> advised him
id 1835 "Democracy is a large and unruly politcal system." politcal -> political
id 1860 "You can buy choes on Amazon" choes -> shoes ?

I stopped looking after id 1860 -- there are likely more that can be found with a spell-checker.

**Comment by wangcunxiang** [comment]
Thanks so much for your carefulness. 
I have to apologize for the typos. 
We did not have enough time to fix all the typos(though we fixed most). 
And it seems that the typos are not too many to affect results? (13/1860, <1%)
For now, the evaluation is still based on the current version of the training set for fairness
But the test set will be carefully checked to ensure that there are no typos.

**Comment by juand-r** [comment]
Ok. Thanks!
